qa/tasks/cephfs/mount.py

```diff
@@ -14,6 +14,32 @@
 
 log = logging.getLogger(__name__)
 
+# This will cleanup the stale netnses, which are from the
+# last failed test cases.
+def cleanup_stale_netnses_and_bridge(remote):
```

Comment:
Let's make this a static method of `CephFSMount`.

Reply:
Sure, done.